### ISSUE TITLE
Post Install script cp failure

### ISSUE THREAD

**Issue by dmjio** [plaidchat/plaidchat#147]
I'm having an issue with libffmpegsumo copying here: https://github.com/plaidchat/plaidchat/blob/master/postinstall.sh#L8

It seems plaidchat isn't copying the ffmpeg shared object built from electron.

```
+ cp node_modules/electron-prebuilt/dist/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
cp: cannot stat 'node_modules/electron-prebuilt/dist/libffmpegsumo.so': No such file or directory
```

Any ideas why? ^

More info here:

```
npm ERR! argv "/nix/store/b3m14jf4swcazpcdhwmqsriv7kgw0k9c-nodejs-4.2.3/bin/node" "/nix/store/j8vavvvsjn4a0r6br82alsklhn6g7g1b-nodejs-4.2.3-npm-3.5.2/bin/npm" "install" "." "--registry=http://notaregistry.662ccf4ced-plaidchat-2.15.0.derp" "--fetch-retry-mintimeout=0" "--fetch-retry-maxtimeout=10" "--fetch-retries=0" "--userconfig=/dev/null" "--nodedir=/nix/store/smbyyv33g5jyfpj9kzm5wk2zycxjnwmg-node-sources" "--production" "--loglevel=verbose"
npm ERR! node v4.2.3
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! plaidchat@2.15.0 postinstall: `./postinstall.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the plaidchat@2.15.0 postinstall script './postinstall.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the plaidchat package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./postinstall.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs plaidchat
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls plaidchat
npm ERR! There is likely additional logging output above.
npm verb exit [ 1, true ]

npm ERR! Please include the following file with any support request:
```

**Comment by twolfson** [comment]
It's possible that `npm@3` is using sharing `electron-prebuilt` in a higher level directory which is leading to the path not being resolved. I will whip up a test branch that uses `require.resolve` instead.

**Comment by twolfson** [comment]
Alright, I have made a test branch:

https://github.com/plaidchat/plaidchat/compare/plaidchat:047acf3...plaidchat:92f6b17

To try it out, please install it via:

```bash
npm install -g git://github.com/plaidchat/plaidchat.git#92f6b17
```

**Comment by twolfson** [comment]
Any feedback on whether the test branch worked or not?

**Comment by elreydetoda** [comment]
Yes this worked, I don't know if this guy ever got back with you but I had the same problem and then downloaded the test branch and it worked properly with no errors on the install.

**Comment by twolfson** [comment]
Great, I will set up a PR for this. Thanks!

**Comment by elreydetoda** [comment]
No problem, thanks for updating it!

**Comment by twolfson** [comment]
We have landed the PR in `2.15.1`. Thanks to everyone for contributing to patch this issue =)
